libretro/libretro.cpp

```diff
@@ -850,15 +850,6 @@ static void check_variables(CoreParameter &coreParam)
          g_Config.bTextureBackoffCache = true;
    }
 
-   var.key = "ppsspp_retain_changed_textures";
```

Comment:
I think this too:
https://github.com/hrydgard/ppsspp/blob/74ac90385f610133976c9cc6c48611cd0c3a872b/libretro/libretro_core_options.h#L431

Not really sure why it has to be in an h file...

-[Unknown]